other/median_of_two_arrays.py

```diff
@@ -0,0 +1,18 @@
+# This code finds the median of two arrays, even if they are not sorted initially
+def find_median_arrays(nums1, nums2):
+    list3 = nums1 + nums2
+    list3 = nums1 + nums2
```

Comment:
```suggestion
    all_numbers = nums1 + nums2
```
Remove duplication and use a more self-documenting variable name.